src/models/permissions.js

```diff
@@ -0,0 +1,20 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+'use strict';
+
+module.exports = (sequelize, DataTypes) => {
+  const Permission = sequelize.define('Permissions', {
+    name: { type: DataTypes.STRING, primaryKey: true }
+  }, {
+    timestamps: false,
+    classMethods: {
+      associate: db => {
+        Permission.belongsToMany(db.Clients, { through: 'ClientPermissions' });
```

Comment:
yeah, I'm fine with `belongsToMany`, but maybe it's not needed to be bidirectional (I mean: both here and in the definition for `Clients`). Adding it here will add "addClient" and such methods to Permission, and I don't think it's useful. I think only `addPermission` and such in Clients are useful.